config/kbin_routes/magazine_api.yaml

```diff
@@ -1,7 +1,7 @@
-# Create an article entry in a magazine
-api_magazine_entry_create_article:
-  controller: App\Controller\Api\Entry\MagazineEntryCreateApi::article
-  path: /api/magazine/{magazine_id}/article
+# Create a thread entry in a magazine
+api_magazine_entry_create_athread:
+  controller: App\Controller\Api\Entry\MagazineEntryCreateApi::thread
+  path: /api/magazine/{magazine_id}/thread
```

Comment:
changing an API path should never be done... You could mark the old path as deprecated and add a new API route at this new path and reference the new one. If you did this change suddenly on your server running latest main, an app (e.g. Interstellar) would not be able to create a thread anymore

Reply:
Correct. We might want to create a new API version number. Also a reason I targeted v1.8.0 version for Mbin.

Since Interstellar is the only client thus far, we could inform them about this backwards incompatible change for now.

Reply:
I really don't want to introduce any unnecessary breaking changes. I understand the need to get rid of confusing nomenclature, but I think we should keep the old endpoint. It just doesn't hurt

Reply:
Sure ok, we will make it backwards compatible for now (and later remove it I suppose).